internal/accountnotifier/account_notifier.go

```diff
@@ -315,15 +315,23 @@ func (n *AccountNotifier) Reconcile(ctx context.Context, ri *reconcile.ResourceI
 			}
 
 			encodedReencryptedPassword := base64.StdEncoding.EncodeToString(unarmoredReencryptedPassword)
-			secretMap := make(map[string]interface{})
-			secretMap["console_url"] = desired.Secret.ConsoleURL
-			secretMap["encrypted_password"] = encodedReencryptedPassword
-			secretMap["account"] = desired.Secret.Account
-			secretMap["user_name"] = desired.Secret.Username
 
-			_, err = n.vault.WriteSecret(fmt.Sprintf("%s/%s_%s", n.vaultExportPath, desired.Secret.Username, desired.Secret.Account), secretMap)
+			type outPutSecret struct {
```

Comment:
nit: for consistency with methods like `GetObjectOutput`
```suggestion
			type outputSecret struct {
```